class_exercises/game.js

```diff
@@ -0,0 +1,43 @@
+
+
+
+Math.floor(Math.random() * 20) + 1;
+//-create a varible that generates a random number between 1 and 10
+
+
+//create a varible that holds the result of the prompt of 
+//asking someone what number they want to guess
+
+//-wrong (the guess does NOT equal the random number)
+//-right (the guess DOES equal the random number)
+
+function guessingGame(){
+
+var randomNumber = Math.floor(Math.random() * 10) + 1;
+//console.log(randomNumber);
+var guess = prompt('guess a number between 1 and 10!');
+console.log(guess);
+if(guess=null){
```

Comment:
one equal sign (=) is used only for variable _assignment_. Here, we need to use a comparison operator, == or ===.